Cat2Cat/main.m

```diff
@@ -141,7 +144,13 @@ int main(int argc, const char * argv[])
            description:@"Asset catalog(s)"
      blockWithArgument:^(NSString *value) {
          [rawAssetCatalogs addObject:value];
-     }];
+    }];
+    [options addOption:"method-name-prefix"
+                  flag:'m'
+           description:@"Prefix for method names (defaults to ac)"
+     blockWithArgument:^(NSString *value) {
+        methodNamePrefix = [value copy];
+    }];
```

Comment:
I wanted to be able to perform a check for nil on the variable. BRLOptionParser didn't seem to be happy if I passed in a pointer to nil. A nil check was needed to bypass the validation check, if the parameter wasn't specified.